lucille-plugins/lucille-jlama/src/test/java/com/kmwllc/lucille/jlama/stage/JlamaEmbedTest.java

```diff
@@ -0,0 +1,160 @@
+package com.kmwllc.lucille.jlama.stage;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.when;
+
+import com.github.tjake.jlama.model.AbstractModel;
+import com.github.tjake.jlama.model.ModelSupport;
+import com.github.tjake.jlama.safetensors.DType;
+import com.github.tjake.jlama.safetensors.SafeTensorSupport;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.io.File;
+import com.kmwllc.lucille.stage.StageFactory;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.MockedStatic;
+
+public class JlamaEmbedTest {
+
+  private StageFactory factory = StageFactory.of(JlamaEmbed.class);
+  public Config defaultConfig;
```

Comment:
should be private

Reply:
changed to private in [27bd5f3](https://github.com/kmwtechnology/lucille/pull/201/commits/27bd5f36fae2a778926fe2c57c08df1b49bed931)